streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamAggProcessorSupplier.java

```diff
@@ -16,12 +16,17 @@
  */
 package org.apache.kafka.streams.kstream.internals;
 
+import org.apache.kafka.common.utils.Bytes;
 import org.apache.kafka.streams.processor.api.ProcessorSupplier;
 
 public interface KStreamAggProcessorSupplier<KIn, VIn, KAgg, VAgg> extends ProcessorSupplier<KIn, VIn, KAgg, Change<VAgg>> {
 
     KTableValueGetterSupplier<KAgg, VAgg> view();
 
+    default KTableValueGetterSupplier<Bytes, byte[]> rawView() {
```

Comment:
This has only a default to make the code compile -- we should remove a default to force all classes to provide a proper implementation -- seems no test fails, what means we actually have tests to add for case that should fail with NPE now.